prime-router/src/main/kotlin/history/db/SubmitterDatabaseAccess.kt

```diff
@@ -216,14 +216,11 @@ class SubmitterDatabaseAccess(val db: DatabaseAccess = BaseEngine.databaseAccess
                 DSL.count().`as`(SubmitterTable.SUBMITTER.TEST_RESULT_COUNT),
                 DSL.value(SubmitterType.PROVIDER.name).`as`(SubmitterTable.SUBMITTER.TYPE),
                 DSL.`when`(
-                    CovidResultMetadata.COVID_RESULT_METADATA.ORDERING_FACILITY_CITY.isNotNull,
-                    CovidResultMetadata.COVID_RESULT_METADATA.ORDERING_FACILITY_CITY
-                ).`when`(
-                    CovidResultMetadata.COVID_RESULT_METADATA.ORDERING_FACILITY_COUNTY.isNotNull,
-                    CovidResultMetadata.COVID_RESULT_METADATA.ORDERING_FACILITY_COUNTY
+                    CovidResultMetadata.COVID_RESULT_METADATA.ORDERING_PROVIDER_COUNTY.isNotNull,
```

Comment:
While fixing the null issue, I realized that I had flipped these between the provider and facility.  Impact seemed minimal since it appears that this will likely be same for provider/facility.